lib/jekyll/site_data/file.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module SiteData
+    class File
```

Comment:
There are no unit tests for this anywhere – is that intentional? I would be happy to see simple unit tests for creating an instance and checking equality, where you have some custom code.